src/main/java/io/codelex/exceptions/Exercise2.java

```diff
@@ -0,0 +1,36 @@
+package io.codelex.exceptions;
+
+public class Exercise2 {
+
+    public static void main(String[] args) {
+        methodA(25, 5);
+
+    }
+
+    public static void methodC(int number, int div) throws ArithmeticException {
+        try {
+            System.out.println(number + " / " + div + " = " + number / div);
+            System.out.println("Method C ");
+        } catch (ArithmeticException e) {
+            System.out.println("You can't divide by 0");
```

Comment:
`throw object to its caller` 
kas būtu `throw e`, izprintē info par exception tajā vietā un tad izmet tālāk nākamaji metodei apstrādāt